pydantic_extra_types/country.py

```diff
@@ -65,7 +63,7 @@ def _index_by_short_name() -> Dict[str, CountryInfo]:
 
 @lru_cache()
 def _index_by_official_name() -> Dict[str, CountryInfo]:
-    return {country.official_name: country for country in _countries() if country.official_name is not None}
+    return {country.official_name: country for country in _countries() if country.official_name}
```

Comment:
It was not possible to get `None`, since `official_name` only returns `str`.

Reply:
👍🏻